### PR TITLE
Update magic link copywriting

### PR DESCRIPTION
**Pull request by IniZio**
ref: #2748

- [x] based on #2857

### Screens

**Setup authenticator**
Screen Shot 2023-01-31 at 11 18 08

**Waiting for email verification**
Screen Shot 2023-01-31 at 11 18 37

**Approval Screen**
Screen Shot 2023-01-31 at 11 21 37

**Success, close window screen**
Screen Shot 2023-01-31 at 11 25 59

**Success at original session screen**
Screen Shot 2023-01-31 at 11 26 21

**Login**
Screen Shot 2023-01-31 at 11 27 08

**Resend status**
Screen Shot 2023-01-31 at 11 28 22

**Approval Screen**
Screen Shot 2023-01-31 at 11 29 19

**Email link verification**

**Email for verify magic link as new authenticator**
Screen Shot 2023-01-31 at 11 58 36

**Email for verify magic link to login**
Screen Shot 2023-01-31 at 11 28 49

### CODE REVIEW

**Comment by fungc-io** [comment]
**Email for verify magic link as new authenticator**
Line height is too tight for the heading in the email body

image

**Comment by IniZio** [comment]
@fungc-io Thanks! Updated

**Comment by carmenlau** [comment]
Need update to remove the duration in the email

**Comment by IniZio** [comment]
Also updated to base on #2857

File: resources/authgear/templates/en/web/verify_magic_link.html [hunk 1]
```diff
@@ -43,7 +50,13 @@ <h1 class="m-0 primary-txt text-center text-xl font-bold">{{ template "verify-ma
...
 
 {{ else }}
-<div class="text-sm break-words primary-txt text-center">{{ template "verify-magic-link-matched-description" }}</div>
+{{ if $.ClientName }}
+	<div class="mb-3 text-sm break-words primary-txt text-center whitespace-pre-wrap">{{ template "verify-magic-link-matched-description" (dict "AppOrClientName" $.ClientName) }}</div>
+	{{ else }}
+	{{ $appName := ($.Translations.RenderText "app.name" nil) }}
+	<div class="mb-3 text-sm break-words primary-txt text-center whitespace-pre-wrap">{{ template "verify-magic-link-matched-description" (dict "AppOrClientName" $appName) }}</div>
+	{{ end }}
+</div>
```
Comment (carmenlau) [inline resources/authgear/templates/en/web/verify_magic_link.html:59]: 1. The indentation is wrong.
2. Unexpected `</div>` tag?
3. In the email, we are using the app name. Keep it simple, maybe we also always use the app name here?

**Comment by carmenlau** [comment]
Thanks for rebasing! I merged it via 0045b30

**Pull request closed without merging**